libraries/AP_RangeFinder/AP_RangeFinder_JRE_Serial.cpp

```diff
@@ -0,0 +1,136 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "AP_RangeFinder_JRE_Serial.h"
+
+#if AP_RANGEFINDER_JRE_SERIAL_ENABLED
+
+#include <GCS_MAVLink/GCS.h>
+
+#define FRAME_HEADER_1   0x52  // 'R'
```

Comment:
I think it would be better to define 'R' and 'A' if they are indicated by comments.

Reply:
I corrected definition of "R" and "A".